plugin/fzf.vim

```diff
@@ -537,6 +549,15 @@ function! s:execute(dict, command, use_height, temps) abort
       call jobstart(cmd, fzf)
       return []
     endif
+  elseif has('win32unix')
+    if $TERM ==# 'cygwin'
```

Comment:
I think this branch can be removed now as the new binary will unset TERM if TERM == cygwin, so line 552 becomes `elseif has('win32unix') && $TERM != 'cygwin'`.